server/utils/is-protected-entity.js

```diff
@@ -2,6 +2,7 @@
 
 const isProtectedEntity = (entity, rules) =>
   rules.some((rule) => {
+    if(rule.length !== 3) return false
```

Comment:
@BorysShulyak Could you please use curly braces around `return false;` and put a blank line between `}` and the next statement?